raiden/api/rest.py

```diff
@@ -2,22 +2,23 @@
 from flask_restful import Api, abort
 
 from webargs.flaskparser import parser
-
-from raiden.api.encoding import (
+from pyethapp.jsonrpc import address_encoder
```

Comment:
please, group the third-party imports and add the coding line `# -*- coding: utf-8 -*-`